src/all-transfers/index.ts

```diff
@@ -109,8 +113,8 @@ async function main() {
       // skip invalid logs
       if (
         log == undefined ||
-        log.indexed.length < 2 ||
-        log.body.length < 1 ||
+        log.indexed.length == 2 ||
```

Comment:
is this intentional? Seems like it changes the logic

Reply:
Yeah, you are right it breaks erc20... But I am focussing on ETH at the moment. I should comment it out temporarily :)

Reply:
I should do `!= 2` and `!=1`